src/reactIntl.js

```diff
@@ -0,0 +1,10 @@
+export const intl = {
+	formatMessage: jest.fn(trn => trn.defaultMessage || 'formatted message'),
+	formatDate: jest.fn(date => date.toString() || 'formatted date'),
+	formatTime: jest.fn(time => time || 'formatted time'),
```

Comment:
this should also use `toString()` https://github.com/yahoo/react-intl/wiki/API#formattime